app/models/mixins/archived_mixin.rb

```diff
@@ -2,11 +2,14 @@ module ArchivedMixin
   extend ActiveSupport::Concern
 
   included do
+    scope :deleted, -> { where(:deleted => true) }
+    scope :not_deleted, -> { where(:deleted => false) }
```

Comment:
Right, I am using scopes in the relation now, so ems.container_groups.unscope(:active) should work?

I think all we need then is to add another scope to the ArchivedMixin. (We can add those in another PR). You got me a bit confused with the ```ContainerGroup.all``` vs. ```ContainerGroup.all```. Those should always return all records from a table.